bin/mana_launch.py

```diff
@@ -42,6 +43,9 @@
   sys.exit(1)
 
 # Special arguments
+if "--gdb" in dmtcp_flags:
+  gdb = True
+  dmtcp_flags.remove("--gdb")
```

Comment:
This is minor, but for style, show about move the `gdb = False` down here with:
```
else:
  gdb = False
```

Then the code is local.  I hate having to search multiple places in the code to understand what it does.

Reply:
The `gdb` variable is initialized to `False` at the beginning of the script. So we don't need another else clause.